parmed/amber/_amberparm.py

```diff
@@ -1996,6 +1998,31 @@ def _add_missing_13_14(self, ignore_inconsistent_vdw=False):
 
     #===================================================
 
+    def _get_atom_dict_for_alternate_labels(self, residues):
+        from collections import defaultdict
+
+        atom_dict = defaultdict(list)
+        for resid, residue in enumerate(residues):
+            atom_dict[resid] = defaultdict(list)
+            for atom in residue.atoms:
+                atom_dict[resid][atom.name].append(atom)
+        return atom_dict
```

Comment:
> Move the import to the top of the file.

just deleted that line since the import is already at the top.

> Also, atom_dict looks to me like it should not be a dict at all, but just a simple list:

ah, I took my code from phenix-amber code. In that case, only a subset of residues having alternative atoms are used. I just updated to your suggestion.

> Also, you can use self.residues rather than passing in a residues object.

updated.